packages/react-components/source/scss/library/components/_table.scss

```diff
@@ -50,6 +50,12 @@
   background-color: $puppet-n200;
   padding: 11px 16px;
   text-align: left;
+
+  code {
+    @include puppet-type-code-small();
```

Comment:
It seems like it might be a little unusual to generate a table that had a code tag in it, but I updated this since I was already making these changes in the tables withing Content